src/solvers/smt2/smt2_dec.cpp

```diff
@@ -44,22 +46,24 @@ std::string smt2_dect::decision_procedure_text() const
 
 smt2_temp_filet::smt2_temp_filet()
 {
-  temp_out_filename=get_temporary_file("smt2_dec_out_", "");
+  temp_problem_filename = get_temporary_file("smt2_dec_problem_", "");
```

Comment:
Could this cleanup (more on this below) please be put in a separate commit? It's surely a very good idea, but it renders (the rather important) git history useless.

Reply:
Ok, split